grammar/alda.bnf

```diff
@@ -22,7 +22,8 @@ chord                   = (note | rest) subchord+
 note                    = pitch duration? <ows> slur?
 rest                    = <"r"> duration? <ows>
 
-pitch                   = !name #"[a-g][+-]*"
+pitch                   = pitch-notation
+pitch-notation          = !name #"[a-g][+-=]*"
```

Comment:
could change THIS to `<pitch-notation>`, so that the contents of the `:pitch-notation` node get lifted up into the next higher level of the parse tree, e.g. into the `:pitch` node